lib/util/cache/repository/types.ts

```diff
@@ -32,6 +33,13 @@ export interface BranchCache {
   upgrades: BranchUpgradeCache[];
 }
 
+export interface OnboardingBranchCache {
+  isOnboarded: boolean;
```

Comment:
Will this object also be stored if isOnboarded is `false`, or only for onboarding repos?

Reply:
Yes, our main use-case is when isOnboarded is `false` to prevent the cloning on re-runs after the onboardingPR is created.

Reply:
So then is this object is present then `isOnboarded` will always be `false`? Maybe redundant then?

Reply:
Yes I will remove it once I add the code to delete the cachedOnboardingBranch.